.test.mk

```diff
@@ -73,10 +73,19 @@ test-release: build run-luajit-test run-test
 
 .PHONY: test-release-asan
 test-release-asan: CMAKE_ENV = CC=clang-11 CXX=clang++-11
+# FIBER_STACK_SIZE=640Kb: The default value of fiber stack size
+# is 512Kb, but some of the tests in test/PUC-Rio-Lua-5.1-test
+# suite in LuaJIT repo (e.g. some cases with deep recursion in
+# errors.lua or pm.lua) have already been tweaked according to the
```

Comment:
Nit: I suppose, that it should be either "some of the tests ... has already been tweaked" either "some tests ... have already been tweaked".
Feel free to ignore.

Reply:
I guess "several tests ... have already been tweaked" is OK.